Plan/velocity/build.gradle

```diff
@@ -16,14 +16,14 @@ dependencies {
 
     compileOnly "com.velocitypowered:velocity-api:$velocityVersion"
     annotationProcessor "com.velocitypowered:velocity-api:$velocityVersion"
-    compileOnly "com.github.ProxioDev.RedisBungee:RedisBungee-API:$redisBungeeProxioDevVersion"
+    compileOnly "com.github.ProxioDev.ValioBungee:RedisBungee-API:$redisBungeeProxioDevVersion"
 
     testImplementation "com.velocitypowered:velocity-api:$velocityVersion"
-    testImplementation "com.github.ProxioDev.RedisBungee:RedisBungee-API:$redisBungeeProxioDevVersion"
+    testImplementation "com.github.ProxioDev.ValioBungee:RedisBungee-API:$redisBungeeProxioDevVersion"
```

Comment:
Could you link me somewhere to give more details about this change

Reply:
https://github.com/ProxioDev/ValioBungee/releases/tag/0.12.3

It's just a GitHub repo name change, but since ValioBungee is built with jitpack, the groupId changed